stream/src/main/scala/org/apache/pekko/stream/javadsl/Flow.scala

```diff
@@ -773,6 +773,9 @@ final class Flow[In, Out, Mat](delegate: scaladsl.Flow[In, Out, Mat]) extends Gr
    * The returned `Iterable` MUST NOT contain `null` values,
    * as they are illegal as stream elements - according to the Reactive Streams specification.
    *
+   * This operator doesn't handle upstream's completion signal, the state kept in the closure can be lost,
```

Comment:
Minor nit, but this sounds better

> This operator doesn't handle upstream's completion signal since the state kept in the closure can be lost. Use [[FlowOps.statefulMap]] instead.